include/swift/Basic/LangOptions.h

```diff
@@ -329,7 +329,7 @@ namespace swift {
 
     /// Whether to enable experimental differentiable programming features:
     /// `@differentiable` declaration attribute, etc.
-    bool EnableExperimentalDifferentiableProgramming = false;
+    bool EnableExperimentalDifferentiableProgramming = true;
```

Comment:
Since upstream is going to be false, should we add `SWIFT_ENABLE_TENSORFLOW`?
```suggestion
    // SWIFT_ENABLE_TENSORFLOW
    bool EnableExperimentalDifferentiableProgramming = true;
```

Reply:
Good catches, thanks! Done in https://github.com/apple/swift/pull/27509/commits/a314d64a301e4efd5d69a4b0404f86dad0470d6b.